lib/storage/platforms/NativeStorage.ts

```diff
@@ -0,0 +1,3 @@
+import SQLiteStorage from '../providers/SQLiteProvider';
```

Comment:
This file has some odd names. The file is called `NativeStorage`, but it exports `SQLiteStorage` while importing `SQLiteProvider`. So... which one is it?

Our naming conventions state that a file should be named for the same thing that it exports so I think that would result in something more like this:

```
import {default as NativeStorage} from '../providers/SQLiteProvider';

export NativeStorage;
```
```